### DISCUSSION TITLE
Result cannot be reproduced

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#146] [BaiduKDDCup2022] [answered]
Our team is SlienceGTeam. We submited the same code as 2022-05-30 23:45. But the result is quite different. Is the evaluation protocols changed after 2022-05-30?

**Comment by ramber1836** [comment, marked as answer]
The score you obtained at 2022-05-30 23:45 is -41.11859. What was the score you obtained with the same code for your recent submission?

**Reply by wang975679801** [reply]
About -41.18. Another submission also has similar reduction, which was -41.14 before and -41.21 now.

**Reply by wang975679801** [reply]
Has this problem been fixed now?

**Reply by Ethan1X** [reply]
Yes, we have the same problem.

**Reply by ramber1836** [reply]
The main difference is shown as below: 
<img width="966" alt="image" src="https://user-images.githubusercontent.com/19561583/172979784-3d9260ce-da58-434b-8c39-4cb58b88a2ea.png">

The current measurement is more reasonable.
